apps/backend/src/modules/notifications/notification.repository.ts

```diff
@@ -82,6 +93,23 @@ class NotificationRepository implements Repository {
 		};
 	}
 
+	public async getUnreadCount(
```

Comment:
We don't need to return all the unread notifications, we only need to return number of them from this method